docs/uvdata_tutorial.rst

```diff
@@ -934,53 +933,55 @@ Note: there is now support for reading in only part of a uvfits, uvh5 or miriad
-  >>> fig, ax = plt.subplots(1, 1)
-  >>> _ = plt.imshow(np.abs(waterfall_data[:,  window_sel]))
-  >>> _ = ax.set_yticks([0, waterfall_times.size - 1])
-  >>> _ = ax.set_yticklabels([waterfall_times[0], waterfall_times[1]])
-  >>> freq_tick_inds = np.concatenate((np.arange(0, freqs_use.size, 16), [freqs_use.size-1]))
-  >>> _ = ax.set_xticks(freq_tick_inds)
-  >>> _ = ax.set_xticklabels([f"{val:.3f}" for val in freqs_use[freq_tick_inds]*1e-9])
-  >>> _ = ax.set_xlabel("Frequency (GHz)")
-  >>> plt.show() # doctest: +SKIP
-  >>> plt.savefig("Images/amplitude_waterfall_spw0.png", bbox_inches='tight')
-  >>> plt.clf()
-
-.. image:: Images/amplitude_waterfall_spw0.png
-    :width: 600
-
 
 UVData: Location conversions
 ----------------------------
 A number of conversion methods exist to map between different coordinate systems
 for locations on the earth.
 
-a) Getting antenna positions in topocentric frame in units of meters
-********************************************************************
+Note that the ``UVData.telescope.location`` attribute is an
+:class:`astropy.EarthLocation` object, so it can be used directly to get to any
+astropy supported coordinate system.
+
+a) Getting antenna positions in East, North, Up (ENU) frame in units of meters
+******************************************************************************
+
+Note that the ENU frame is sometimes referred to as the topocentric frame but
+in many references the topocentric frame has the pole on the axis of rotation
+for the Earth rather than at the local zenith. We just call it the ENU frame for
+clarity.
+
+Use the :meth:`pyuvdata.Telescope.get_enu_antpos` to get the antenna
+positions in the ENU frame. Or use the ``telescope.location`` and
+``telescope.antenna_positions`` attributes (which are ECEF positions relative
+to the ``telescope.location``) with the :meth:`pyuvdata.utils.ENU_from_ECEF`
+utility method.
+
 .. code-block:: python
 
-  >>> # directly from UVData object
+  >>> # directly from Telescope object
   >>> import os
+  >>> from astropy.units import Quantity
   >>> from pyuvdata import UVData
   >>> from pyuvdata.data import DATA_PATH
   >>> data_file = os.path.join(DATA_PATH, 'new.uvA')
-  >>> uvd = UVData.from_file(data_file, use_future_array_shapes=True)
-  >>> antpos, ants = uvd.get_ENU_antpos()
+  >>> uvd = UVData.from_file(data_file)
+  >>> antpos = uvd.telescope.get_enu_antpos()
 
   >>> # using utils
   >>> from pyuvdata import utils
 
   >>> # get antennas positions in ECEF
-  >>> antpos = uvd.antenna_positions + uvd.telescope_location
-
-  >>> # convert to topocentric (East, North, Up or ENU) coords.
-  >>> antpos = utils.ENU_from_ECEF(antpos, *uvd.telescope_location_lat_lon_alt)
+  >>> telescope_ecef_xyz = Quantity(uvd.telescope.location.geocentric).to("m").value
+  >>> antpos = uvd.telescope.antenna_positions + telescope_ecef_xyz
+
+  >>> # convert to East, North, Up (ENU) coords.
+  >>> antpos = utils.ENU_from_ECEF(
```

Comment:
yes, I'll work on that.

Reply:
I'd even be fine with an issue for now and add that in after

Reply:
So looking more at this, it actually does optionally accept an EarthLocation object rather than the lat/lon/alt for the center location, it just wasn't being called that way here in the tutorial. I'll update that.

Reply:
Oh! look at past us, being smart. That'll show how often I go read our own docs.